scripts/build/libc/glibc.sh

```diff
@@ -247,6 +247,10 @@ do_libc_backend_once() {
     [ -n "${CT_TOOLCHAIN_BUGURL}" ] && extra_config+=("--with-bugurl=${CT_TOOLCHAIN_BUGURL}")
 
     touch config.cache
+
+    # Hide host C++ binary from configure
+    echo "ac_cv_prog_ac_ct_CXX=" >>config.cache
```

Comment:
Doesn't this disable creation of C++ headers even if the final toolchain if C++ is enabled in the configuration?